### ISSUE TITLE
Release a official log4j2-jboss-logmanager version

### ISSUE THREAD

**Issue by boris-unckel** [jboss-logging/log4j2-jboss-logmanager#10]
Please release a official log4j-jboss-logmanager version and create a GIT tag for that.
Currently it is necessary to create always a proprietary version and one can't contribute to other projects with this dependency.

[https://mvnrepository.com/artifact/org.jboss.logmanager/log4j2-jboss-logmanager](https://mvnrepository.com/artifact/org.jboss.logmanager/log4j2-jboss-logmanager)

**Comment by jamezp** [comment]
Hi @boris-unckel. Would an Alpha or Beta release be enough?

**Comment by boris-unckel** [comment]
Yes - any release can be referenced. Thank you @jamezp

**Comment by jamezp** [comment]
@boris-unckel I've released 1.0.0.Beta1 of this. My plan is to do some more testing in WildFly, then I'll release a final release if no other changes are expected.

One thing I've looked at is a way to detect a different implementation, e.g. log4j-core, and delegate to that instead. However I'm not sure if that should live in this project or not. The idea was more for WildFly in cases where a user includes log4j-core in their deployment as they may want to use that log manager implementation. Anyway I digress and am somewhat thinking out loud :)
